packages/calcite-components/src/components/alert/AlertManager.ts

```diff
@@ -0,0 +1,67 @@
+export const alertQueueTimeoutMs = 300;
+
+export default class AlertManager {
+  // --------------------------------------------------------------------------
+  //
+  //  Private Properties
+  //
+  // --------------------------------------------------------------------------
+
+  private registeredElements: HTMLCalciteAlertElement[] = [];
```

Comment:
In the future, this could be a map 

```ts
 private registeredElementsMap: Map<string, HTMLCalciteAlertElement[]> = new Map([
    ["default", []],
  ]);
```

This would allow us to have alert `groups` by adding a `group` property to `Alert`. Then only certain alerts would be grouped with other ones.